apps/api/src/country/dto/create-country.dto.ts

```diff
@@ -0,0 +1,22 @@
+import { ApiProperty } from '@nestjs/swagger'
+import { City } from '@prisma/client'
+import { Expose } from 'class-transformer'
+import { IsOptional, IsString } from 'class-validator'
+
+@Expose()
+export class CreateCountryDto {
+  @ApiProperty()
+  @Expose()
+  @IsString()
+  name: string
+
+  @ApiProperty()
+  @Expose()
+  @IsString()
```

Comment:
We can be more specific here and restrict to iso code `@IsISO31661Alpha2()`

Check https://github.com/typestack/class-validator#validation-decorators